### PR TITLE
Feature/remote active

### PR DESCRIPTION
**Pull request by ethani149**

### CODE REVIEW

File: webserver_pkg/webserver_pkg/vehicle_control.py [hunk 1]
```diff
@@ -168,29 +112,70 @@ def api_manual_drive():
...
-    msg.angle = -1.0 * get_categorized_manual_angle(angle)
-    categorized_throttle = get_categorized_manual_throttle(throttle)
-    msg.throttle = -1.0 * get_rescaled_manual_speed(categorized_throttle, max_speed)
+    # msg.angle = -1.0 * get_categorized_manual_angle(angle)
+    # categorized_throttle = get_categorized_manual_throttle(throttle)
+    # msg.throttle = -1.0 * get_rescaled_manual_speed(categorized_throttle, max_speed)
+    speed = webserver_publisher_node.get_speed_value()
     webserver_node.pub_manual_drive.publish(msg)
-    return jsonify({"success": True})
+    return jsonify({"success": True, "speed": speed})
```
Comment (tback123) [inline webserver_pkg/webserver_pkg/vehicle_control.py]: should we add `remoteActive` here as well?

File: webserver_pkg/webserver_pkg/webserver_publisher_node.py [hunk 2]
```diff
@@ -137,6 +137,15 @@ def __init__(self):
             10)
         self.subscription_gps_speed  # prevent unused variable warning
 
+        #TODO: Change Topic
+        self.subscription_remote_active = self.create_subscription(
+            Bool,
+            '/remote/enable',
```
Comment (tback123) [inline webserver_pkg/webserver_pkg/webserver_publisher_node.py]: should be `/remote/active`

**Review by tback123 — CHANGES_REQUESTED** [review]

File: webserver_pkg/webserver_pkg/webserver_publisher_node.py (hunk 2 above)
Comment (ethani149) [reply webserver_pkg/webserver_pkg/webserver_publisher_node.py]: Resolved

File: webserver_pkg/webserver_pkg/vehicle_control.py (hunk 1 above)
Comment (ethani149) [reply webserver_pkg/webserver_pkg/vehicle_control.py]: Added Remote Active to Manual Drive endpoint return

**Review by tback123 — APPROVED** [review]

**Pull request merged**